pages/viewPlans/index.js

```diff
@@ -13,22 +16,28 @@ const NewTemplateLink = styled(Link)`
   background-color: yellow;
 `;
 
+const StyledParagraph = styled.p`
+  margin: 0;
+  padding: 1rem 0;
+  text-align: center;
+`;
+
 export default function ViewAllPlans() {
-  const router = useRouter();
   const { data } = useAllPlans();
 
   if (!data) return <p>Loading ...</p>;
 
   return (
-    <div>
-      <p>
-        {`There are`} {data.length}{" "}
-        {data.length === 1 ? "template" : "templates"} {`in your vault`}
-      </p>
+    <OverviewContainer>
+      <StyledParagraph>
+        {data.length === 1
+          ? `There is ${data.length} template in your vault`
+          : `There are ${data.length} templates in your vault`}
+      </StyledParagraph>
       {data.map((plan) => (
```

Comment:
Please make sure to create a list and list items when mapping over an array to be accessible.

Reply:
Ok, will do.